_rules/block-collapsible-3e12e1.md

```diff
@@ -29,7 +29,7 @@ This rule applies to any [HTML web page][].
 
 ## Expectation
 
-For each [block of repeated content][] in each test target, which is before (in the [flat tree][]) at least one node of [non-repeated content after repeated content][], all the following are true:
+For each test target, and each of its [block of repeated content][] which is before at least one node of [non-repeated content after repeated content][] in the [flat tree][], all the following are true:
```

Comment:
> As this is currently phrased I would want it to say `each of its blocks of repeated content` (make blocks plural) which I realize would break the definition tool. Could we say something like `each descendant block of repeated content which is ...`.

Making it plural and adding a new difinition in [block of repeated content]: #block-of-repeated-content 'Definition of Block of Repeated Content'G[blocks of repeated content]: #block-of-repeated-content 'Definition of Block of Repeated Content'